src/cpp/server/ServerPAMAuth.cpp

```diff
@@ -104,10 +104,13 @@ void signIn(const http::Request& request,
 void publicKey(const http::Request&,
                http::Response* pResponse)
 {
-   std::string exp, mod;
-   core::system::crypto::rsaPublicKey(&exp, &mod);
+   std::string publicKey;
+   Error error = core::system::crypto::rsaPublicKey(&publicKey);
+   if (error)
+      LOG_ERROR(error);
+   
    pResponse->setNoCacheHeaders();
-   pResponse->setBody(exp + ":" + mod);
+   pResponse->setBody(publicKey);
    pResponse->setContentType("text/plain");
 }
```

Comment:
We'll have to see where this gets used, and see if we can retool it to take the public key directly.

Reply:
Looks like this is primarily used when a user signs in; this has been updated appropriately in the sign in page.